providers/google/src/airflow/providers/google/cloud/links/base.py

```diff
@@ -19,7 +19,8 @@
 
 from typing import TYPE_CHECKING, ClassVar
 
-from airflow.models import BaseOperatorLink, XCom
+from airflow.models import XCom
+from airflow.sdk.definitions.baseoperatorlink import BaseOperatorLink
```

Comment:
This will need to be a `if airflow3` conditional import

Reply:
Oh for the compat, right!